insurance_custom_drift/dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.7.5
+
+RUN pip3 install pip -U
+
+RUN pip3 install setuptools wheel packaging scikit-learn
+
+RUN pip3 install git+https://github.com/oneconvergence/dkube.git@monitoring-utils
```

Comment:
shouldn't this branch be 3.7? dkube.git@3.7

Reply:
yes, I will correct.